dashboard/src/components/ui/sheet.tsx

```diff
@@ -49,18 +49,21 @@ const sheetVariants = cva(
 
 interface SheetContentProps
   extends React.ComponentPropsWithoutRef<typeof SheetPrimitive.Content>,
-    VariantProps<typeof sheetVariants> {}
+    VariantProps<typeof sheetVariants> {
+      onKeyDown?: (e: React.KeyboardEvent) => void;
```

Comment:
This component already accepts the `onKeyDown` pop, so there is no need to change it.